misc/testerina/modules/report-tools/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ballerina/testerina-report",
-  "version": "2.0.0-Preview1-M3",
```

Comment:
Why are we committing this and the next file?

Reply:
I'll correct it